Formula/tkey-ssh-agent.rb

```diff
@@ -0,0 +1,53 @@
+class TkeySshAgent < Formula
+  desc "SSH agent for use with the TKey security stick"
+  homepage "https://tillitis.se/"
+  url "https://github.com/tillitis/tillitis-key1-apps/archive/v0.0.4.tar.gz"
+  sha256 "b3f3547401159a8a4277a4b9689632699cb6cf5bebde6ba6a093aab80b1e7ea9"
+  license all_of: ["GPL-2.0-only", "CC0-1.0"]
```

Comment:
The repository states `GPL-2.0-only` and the sources for this program also only indicate the GPL. Am I missing something?

```suggestion
  license "GPL-2.0-only"
```

Reply:
There is another license [within the repository](https://github.com/tillitis/tillitis-key1-apps/blob/main/apps/monocypher/LICENSE).
I do not build this formula using that code directly, but there is a pre-built RISC-V binary for the TKey security stick, which uses the code with that license. The formula [downloads](https://github.com/tillitis/tillitis-key1-apps/releases/download/v0.0.4/signer.bin) that RISC-V binary and embeds it in the SSH agent (sometimes the agent needs to update the firmware in the security stick).
Therefore I assumed that I need to state that license as well. Or should I disregard that since a prebuilt product is used from a repository with only GPL 2.0 specified?

Reply:
The pre-compiled signer binary itself is under GPL-2.0-only because it's tainted by the GPL license in the source code, even if it links with a library released under CC0.